packages/console/src/pages/SignInExperience/components/Preview.tsx

```diff
@@ -176,6 +177,12 @@ const Preview = ({ signInExperience, className }: Props) => {
       </TabNav>
       <div
         className={classNames(styles.body, platform === 'desktopWeb' ? styles.web : styles.mobile)}
+        style={conditional(
+          platform === 'desktopWeb' && {
+            // Set background color to match iframe's background color on both dark and light mode.
+            backgroundColor: mode === AppearanceMode.DarkMode ? '#2A2C31' : '#e5e1ec',
+          }
+        )}
```

Comment:
Can't we use CSS?

Reply:
This is a special case, not using css variables, so I use inline style to make it more clear.